### ISSUE TITLE
Entries partial missing

### ISSUE THREAD

**Issue by VojtechKlos** [octobercms/october#5723]
Hello,

I am trying to use "entries" field type in Tailor mixin like this:

```
fields:
    content:
        type: nestedform
        showPanel: false
        form:
            tabs:
                fields:
                    navigation:
                        label: Linked navigation
                        type: entries
                        source: Navigation
                        tab: Nav
```

And when I go to edit the Entry where the field is supposed to be rendered in the backend it throws this exception:
```
The partial '_field_entries' is not found.
~/modules/system/traits/ViewMaker.php line 92
``` 
```
 }
  
        $partialPath = $this->getViewPath($partial);
  
        if (!$partialPath || !File::exists($partialPath)) {
            if ($throwException) {
                throw new SystemException(Lang::get('backend::lang.partial.not_found_name', ['name' => $partial]));
            }
  
            return false;
        }
  
        return $this->makeFileContents($partialPath, $params);
```
OctoberCMS version 3.3.10

Is there something I am missing? (apart from the partial of course :-D)

**Comment by daftspunk** [comment]
Hi @VojtechKlos 

It looks like this is because of the "tabs" definition. Removing it seems to help:

```yaml
fields:
    content:
        type: nestedform
        showPanel: false
        form:
            fields:
                navigation:
                    label: Linked navigation
                    type: entries
                    source: Navigation
                    tab: Nav
```

We've added this issue to our internal tracker.

**Comment by daftspunk** [comment]
This is fixed in v3.3.11
